app/src/main/java/com/infomaniak/mail/ui/newMessage/AiPropositionFragment.kt

```diff
@@ -165,15 +168,18 @@ class AiPropositionFragment : Fragment() {
 
     private fun choosePropositionAndPopBack() = with(aiViewModel) {
 
-        fun applyProposition(subject: String?, content: String) {
+        fun applyProposition(subject: String?, content: String) = with(newMessageViewModel) {
```

Comment:
You use a `with(newMessageViewModel)`only for the liveData.

Reply:
Will be used for a 2nd one in the next PR.